apps/api/src/app/controllers/command/commands.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller, Post, Query, Res } from '@nestjs/common';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import isStringWithNonzeroLength from '../../../lib/utilities/isStringWithNonzeroLength';
+import httpStatusCodes from '../../constants/httpStatusCodes';
+
+@ApiTags('commands')
+@Controller('commands')
+export class CommandController {
+    @ApiOkResponse()
+    @Post('')
+    issueCommand(
+        @Res() res,
+        @Query('type') type: string,
+        @Query('payload') payload: Record<string, unknown>
+    ) {
+        if (!isStringWithNonzeroLength(type))
+            return res
+                .status(httpStatusCodes.badRequest)
+                .send(`Invalid command type: ${JSON.stringify(type)}`);
+
+        console.log(`You issued the command: ${type}`);
+
```

Comment:
Would be good to have some logic to process the command here ;P

Reply:
Great. Now I see where I left off...